voorhoede/head-start
# Feature: Skip Link

Adds skip link component and translation for its text.

# Changes

- skip link is only visible when focused.
- skip link brings user directly to the main content.
- skip link text is localised (seed for translation is moved to #27).
- reasoning behind the skip link is documented in component readme.

# Associated issue

Resolves #24

# How to test

1. Open preview link
2. Verify there is no skip link visible
2. Focus on the browser address bar
3. Hit the tab button
4. Verify that the skip link is focused and visible
5. Hit enter
6. Verify that the focus skipped over the locale selector and is now on the main content
7. Verify that the skip link is no longer visible
8. Repeat the same for all locales and verify the skip link texts are localised

# Checklist

- ~~[ ] I have added/updated tests to prove that my feature works (if not possible please explain why)~~
